app/frontend/courses/review/CoursesReview__Root.res

```diff
@@ -746,7 +746,7 @@ let computeInitialState = () => {
 let pageTitle = (courses, courseId) => {
   let currentCourse = ArrayUtils.unsafeFind(
     course => AppRouter__Course.id(course) == courseId,
-    "Could not find currentCourse with ID " ++ courseId ++ " in CoursesReview__Root",
+    I18n.t("shared.current_course_not_found") ++ courseId ++ " in CoursesReview__Root",
```

Comment:
`current_course_not_found` key is missing in `en.yml`; this change should be reversed.